web/src/main/angular/src/app/core/components/info-per-server/info-per-server-for-filtered-map-container.component.ts

```diff
@@ -100,7 +100,8 @@ export class InfoPerServerForFilteredMapContainerComponent implements OnInit, On
             this.storeHelperService.dispatch(new Actions.ChangeAgentForServerList({
                 agent: this.selectedAgent,
                 responseSummary: this.agentHistogramData['agentHistogram'][this.selectedAgent],
-                load: this.agentHistogramData['agentTimeSeriesHistogram'][this.selectedAgent]
+                load: this.agentHistogramData['agentTimeSeriesHistogram'][this.selectedAgent],
+                responseStatistics: this.agentHistogramData['agentResponseStatistics'][this.selectedAgent]
```

Comment:
Adding `agentResponseStatistics` property is good but if you look up the assigning process for `agentHistogramData` in line 90, you need to add `agentResponseStatistics` property there as well for this process to work well. Now it emits an error because of the missing property.

```
tap(({serverList, agentHistogram, agentTimeSeriesHistogram, agentResponseStatistics, isWas}: INodeInfo | IShortNodeInfo) => {
    this.agentHistogramData = {
        serverList,
        agentHistogram,
        agentTimeSeriesHistogram,
        agentResponseStatistics,
        isWas
    };
})
```

And please add `agentResponseStatistics` in the interface declaration for `INodeInfo` and `IShortNodeInfo` as well.

Reply:
@binDongKim  updated and checked.